packages/dnb-eufemia/src/extensions/payment-card/icons/index.js

```diff
@@ -4,7 +4,7 @@ import BankAxept from './BankAxept'
 import DNB from './DNB'
 import Expired from './Clock'
 import Blocked from './Padlock'
-import NotActive from './CardIn'
+import CardIn from './CardIn'
```

Comment:
```suggestion
import { card_in_medium as CardIn } from '../../../icons'
```
We should delete the `CardIn.tsx` file.